app/react/Documents/components/specs/DocumentSidePanel.spec.js

```diff
@@ -108,6 +109,27 @@ describe('DocumentSidePanel', () => {
         ).toBe('metadata');
       });
     });
+
+    describe('when doc passed has toc', () => {
+      const toc = ['title1', 'title2', 'title3'];
+
+      it('if is a new entity should set toc of the loaded file', () => {
```

Comment:
Please, start all "it" statements with "should".

The way tests are meant to be read is:  `Description`: it `should... `.

In this case, if you concatenate the wordings:
> When doc passed has toc, it if is a new entity...

This is not correct.  What we want is:
> When doc passed has toc, it should set TOC of the loaded file (if doc is a new entity)

So, the 'it' should read: "should set the TOC of the loaded file (if doc is a new entity).

Ideally, "if"s are actually extra describes:
```
describe('when doc passed has ToC')
  describe('when doc is a new entity')
     it('should set ...')
```

This is the "preferred" approach, but sometimes that means nesting too much.  So, you decide here which approach you prefer, but every "it" should start with "should", no matter what.